### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cotyembry**
I tried to improve on what was already there, hope it helps!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/RedHat-Israel/ROSE.svg?branch=master)](https://travis-ci.org/RedHat-Israel/ROSE)
 
 This project is a game that has been developed in order to help teach kids python.
-The students need to code the behavior of the car to achieve the best score.
+The students need to code the behavior of a car to achieve the best score.
```
Comment (nirs) [inline README.md]: Looks good.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,16 @@ Here is video of a race (running code from students):
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas.
+They need to calculate the best path to take to avoid the pitfalls.
+The participants will also have opportunities to collect bonus points.
```
Comment (nirs) [inline README.md]: I don't think this change is better then the original text. Can you explain what was wrong with the original text?

File: README.md [hunk 3]
```diff
@@ -12,15 +12,16 @@ Here is video of a race (running code from students):
...
 The cars move autonomously on the screen within the race track game with no interference
 from the students. No joystick or mouse would be used.
  
 In order to control the car movements, the students needs to implement a 'driver'.
-This code is controlling the car and will decide what will be the next action of the car.
+This code, when ran, will control the car and will hold the logic to take the appropriate action based on each different type of obstacle encountered.
 
-For each type of obstacles there is a different action, and different points.
+For each type of obstacle there is a different action, and different points.
```
Comment (nirs) [inline README.md]: A better change was suggested in another PR.

File: README.md [hunk 4]
```diff
@@ -12,15 +12,16 @@ Here is video of a race (running code from students):
...
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas.
+They need to calculate the best path to take to avoid the pitfalls.
+The participants will also have opportunities to collect bonus points.
 The cars move autonomously on the screen within the race track game with no interference
 from the students. No joystick or mouse would be used.
  
 In order to control the car movements, the students needs to implement a 'driver'.
-This code is controlling the car and will decide what will be the next action of the car.
+This code, when ran, will control the car and will hold the logic to take the appropriate action based on each different type of obstacle encountered.
```
Comment (nirs) [inline README.md]: I'm not sure this is better, the original text seems more clear to me.

File: README.md [hunk 5]
```diff
@@ -31,7 +32,7 @@ To install the dependencies, run:
 
     pipenv install
 
-Or in the old-fashioned way
+Or the old-fashioned way
```
Comment (nirs) [inline README.md]: Looks good.

File: README.md [hunk 6]
```diff
@@ -46,7 +47,7 @@ Start the server on some machine:
 
 Open a browser at http://\<server-address\>:8880 to view and control the game.
 
-Start up to 2 clients:
+Start up two clients:
```
Comment (nirs) [inline README.md]: Looks good.

**Comment by cotyembry** [comment]
I would rebase it, if I was that competent with git

File: README.md (hunk 2 above)
Comment (cotyembry) [reply README.md]: Its separated away from using the semicolon which is awkward with the and coming after it. just my opinion, no worries

**Comment by nirs** [comment]
@cotyembry thanks for your work! I merged most of the changes in 4bd87e2d9e447.

**Comment by cotyembry** [comment]
Thanks bro

"Trust in the LORD with all your heart; do not depend on your own understanding. Seek His will in all you do, and He will show you which path to take."
-‭‭Proverbs‬ ‭3:5-6
http://bible.com/116/pro.3.5.nlt

**Comment by nirs** [comment]
@cotyembry :+1:

**Pull request closed without merging**